### DISCUSSION TITLE
Trilium not launch windows verions

### DISCUSSION THREAD

**Discussion by BrettBass** [TriliumNext/Trilium#2538] [Q&A] [answered]
Trilium fails to launch, I downloaded the latest version for windows. Tried running as administrator, gave it firewall perms, redownloaded it. Can't get the executable to launch

**Comment by zadam** [comment, marked as answer]
Hi, are you upgrading or doing a new installation?

Could you please provide some logs (from `C:\Users\[user]\AppData\Roaming\trilium-data\log`) or some other error output?

**Reply by BrettBass** [reply]
It was one-drive, I don't use one-drive but since it was full it was preventing me from launching.

**Reply by BrettBass** [reply]
After removing one drive I deleted trilium-data

**Reply by zadam** [reply]
Note that using One drive to sync the data will [sooner or later corrupt your data](https://github.com/zadam/trilium/wiki/FAQ#can-i-use-dropbox--google-drive--onedrive-to-sync-data-across-multiple-computers).
